torch/signal/windows/windows.py

```diff
@@ -759,3 +761,64 @@ def general_hamming(M,
                           layout=layout,
                           device=device,
                           requires_grad=requires_grad)
+
+
+@_add_docstr(
+    r"""
+Computes the minimum 4-term Blackman-Harris window according to Nuttall.
+
+{normalization}
+
+Arguments:
+    {M}
+
+Keyword args:
+    alpha (float, optional): the window coefficient. Default: 0.54.
+    {sym}
+    {dtype}
+    {layout}
+    {device}
+    {requires_grad}
+
+Returns:
+    {return}
```

Comment:
I was thinking when looking at the docs, do you think we should just remove the `Returns` field from all the windows? It adds quite a bit of vertical space (two new fields) and it provides little-to-none new info.